iina/uk.lproj/Localizable.strings

```diff
@@ -217,5 +217,5 @@
 "touchbar.rewind" = "Перемотати назад";
 "touchbar.seek" = "Перемотати";
 "touchbar.time" = "Часова позиція";
-"touchbar.toggle_pip" = "Перемкнути «Кадр в кадрі»";
+"touchbar.toggle_pip" = "Перемкнути режим «Картинка в картинці»";
```

Comment:
Wrong. The red one is used in official Ukrainian localisation of macOS apps (eg., Safari web browser).
<img width="292" alt="Screenshot 2019-05-06 16 38 04" src="https://user-images.githubusercontent.com/36358174/57229233-3ca09f00-701e-11e9-9104-12819044c9e9.png">